src/components/TaskList/TaskList.jsx

```diff
@@ -1,11 +1,37 @@
 import * as React from 'react';
```

Comment:
Depending on what version of vite you're using, you don't need to import react, unless you need something specific from it, e.g useState, in that case you would just do `import { useState} from 'react'`